dockerscripts/healthcheck.sh

```diff
@@ -19,7 +19,8 @@ set -x
 
 _init () {
     scheme="http://"
-    address="127.0.0.1:9000"
+    port="$(netstat -npta | grep 1/minio | grep LISTEN | awk '{ print $4 }' | cut -d: -f4)"
```

Comment:
Good bug fix - we need to actually expand the scope a bit, following code would work fine.

```
address=$(netstat -nplt 2>/dev/null | awk ' /(.*\/minio)/ { gsub(":::","127.0.0.1:",$4); print $4}')
```

The reason is we need to support two different type of scenarios here

- Listening on tcp6 by default i.e and user chooses a different port. 
- Listening on tcp only i.e user provided `--address` the right IP or hostname